src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import "../styles/blocks/variables.scss";
+@import "../styles/blocks/header.scss";
+@import "../styles/blocks/cards.scss";
+@import "../styles/blocks/stars.scss";
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
 }
```

Comment:
```suggestion

```
Consider creating a separate file for basic styling and moving these styles here, to keep your main file clean